CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 
 ### 3.x (Next)
 
+* [#49](https://github.com/dblock/ios-snapshot-test-case-expecta/pull/49): Migrate CI to github actions - [@brainbicycle](https://github.com/brainbicycle)
```

Comment:
Nits: 

1. GitHub is spelled GitHub. You can abbreviate GHA.
2. Put a period at the end of this line.